src/pixel_map.rs

```diff
@@ -52,7 +52,9 @@ impl FromProto<&ImageData> for ByteMap {
 		let size = grid.get_size();
 		Array2::from_shape_vec(
 			(size.get_y() as usize, size.get_x() as usize),
-			grid.get_data().to_vec(),
+			grid.get_data().iter().copied().collect(),
+			// below causes sc2 to fail
```

Comment:
this line just copies slice to new vec, see: https://doc.rust-lang.org/std/primitive.slice.html#method.to_vec
not the cause, bug fixed here: 72a716828aa290918f766f759ae498e8d2e857b7

Reply:
Fixed! (And, ya, I wasn't sure what was causing what, so I am probably way off, haha)